api.go

```diff
@@ -2,13 +2,17 @@
 
 package termbox
 
-import "github.com/mattn/go-runewidth"
-import "fmt"
-import "os"
-import "os/signal"
-import "syscall"
-import "runtime"
-import "time"
```

Comment:
Oops sorry about the formatting changes. I'll fix it.

Reply:
your change is more correct IMO

Reply:
Yeah, doesn't look like upstream used `gofmt` (or `gofmt`'s behaviour changed since they did?).

I was thinking it'd be easier to merge in upstream changes if I changed this back to how it was, but it feels like everyone from this point on would have to keep reverting their changes if we did that! I'll leave it as it is then :+1: